crates/router/src/core/payments/operations/payment_response.rs

```diff
@@ -96,7 +96,9 @@ impl<F: Send + Clone> PostUpdateTracker<F, PaymentData<F>, types::PaymentsAuthor
         let customer_id = payment_data.payment_intent.customer_id.clone();
         let save_payment_data = tokenization::SavePaymentMethodData::from(resp);
         let profile_id = payment_data.payment_intent.profile_id.clone();
-        let is_mandate = &resp.request.setup_mandate_details.is_some();
+        let connector_mandate = &resp.request.setup_mandate_details.is_some();
```

Comment:
Check for customer_acceptance here instead of setup_future_usage

Reply:
Are you suggesting to have a check for `customer_acceptance` instead of `setup_mandate_details` for relating a payment to be `connector_mandate` (a mandate that we is generated on the connector's end) ?
```suggestion
        let connector_mandate = &resp.request.customer_acceptance.is_some();
```

Something like this and are we sure that setup_mandate_details is not required for this flow?

Reply:
Yes